src/reader.ts

```diff
@@ -402,26 +403,48 @@ export default class D2Reader {
   };
 
   private isTTSIncrementable(
-    incremental: UserSettingsIncrementable | TTSIncrementable
+    incremental:
+      | UserSettingsIncrementable
+      | TTSIncrementable
+      | MediaOverlayIncrementable
   ): incremental is TTSIncrementable {
     return (
       incremental === "pitch" ||
       incremental === "rate" ||
       incremental === "volume"
     );
   }
+  private isMOIncrementable(
+    incremental:
+      | UserSettingsIncrementable
+      | TTSIncrementable
+      | MediaOverlayIncrementable
+  ): incremental is MediaOverlayIncrementable {
+    return (
+      incremental === "pitch" ||
+      incremental === "rate" ||
+      incremental === "volume"
```

Comment:
It looks like the `MediaOverlayIncrementable` is only for `"volume"` though right?

Reply:
Yes but there will be rate at some point as well.

Reply:
I think we should keep it to just what is implemented for now otherwise we could get errors. What this function is saying is that the passed in value _definitely_ conforms to the `MediaOverlayIncrementable` type, but in this case we know that is not true because neither `"pitch"` nor `"rate"` conform to that type currently

Reply:
i realized i didn't remove the once that are not implemented, just removed them.